content/en/continuous_integration/setup_tests/swift.md

```diff
@@ -537,7 +537,7 @@ Additional Git configuration for physical device testing:
 
 ## Manual testing API
 
-If you use XCTests with your Swift projects, the `DatadogSDKTesting` framework automatically instruments them and sends the results to the Datadog backend. If you don't use XCTest, you can instead use the Swift/Objective-C manual testing API, which also reports test results to the backend. 
+If you use XCTests with your Swift projects, the `DatadogSDKTesting` framework automatically instruments them and sends the results to the Datadog backend. If you don't use XCTest, you can instead use the Swift/Objective-C manual testing API, which also reports test results to the backend.
```

Comment:
my editor automatically removes trailing spaces, let me know if this is a problem